README.md

```diff
@@ -32,10 +32,36 @@ In addition to the in-library decoding support, you may use the included `plcras
 `plcrashutil convert --format=iphone example_report.plcrash | symbolicatecrash`
 Future library releases may include built-in re-usable formatters, for outputting alternative formats directly from the phone.
 
-## Usage
+## Integration
 
 PLCrashReporter can be added to your app via [CocoaPods](https://guides.cocoapods.org/using/using-cocoapods.html), [Carthage](https://github.com/Carthage/Carthage#quick-start), [Swift Package Manager](https://developer.apple.com/documentation/xcode/adding_package_dependencies_to_your_app), or by manually adding the binaries to your project.
 
+### Integration via Cocoapods
+
+1. Add the following line to your `Podfile`:
+```ruby
+pod 'PLCrashReporter'
+```
+1. Run `pod install` to install your newly defined pod and open the project's `.xcworkspace`.
+
+### Integration via Swift Package Manager
+
+1. From the Xcode menu, click **File** > **Swift Packages** > **Add Package Dependency**.
+1. In the dialog that appears, enter the repository URL: https://github.com/microsoft/plcrashreporter.git.
+1. In Version, select Up to Next Major and take the default option.
+
+### Integration via Carthage
+
+1. Add the following line to your `Cartfile`:
+```ruby
+github "microsoft/plcrashreporter"
+```
+2. Run `carthage update` to fetch dependencies.
```

Comment:
Same. Please harmonize numbering across the list. (auto-incrementing works on GH)

Reply:
It actually worked after adding indents to the code blocks.